gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -198,8 +211,40 @@ def __get_initial_values(self, wTi_initial: List[Optional[PosePrior]]):
                 initial.insertPoint3(i, wTi.value.translation())
         return initial
 
+    def augment(
```

Comment:
Add docstring

Reply:
Ack. What about renaming the function to `augment_measurement_with_priors`?